models/public/midasnet/model.yml

```diff
@@ -23,15 +23,15 @@ description: >-
   The model output is an inverse depth map that is defined up to an unknown scale factor.
 task_type: monocular_depth_estimation
 files:
-  - name: midas_net.py
+  - name: models/midas_net.py
```

Comment:
I would prefer `model`, since there is only one model in it, not counting base blocks.